### ISSUE TITLE
Installation issue on Mac book pro/ VS Code

### ISSUE THREAD

**Issue by jw235** [morris-lab/CellOracle#181]
Hey there, 
I pip installed cell oracle on my vs code 3.8.18 environment on an apple m2 silicon and I am not able to import cell oracle in my Jupyter notebook cell. Did anyone experience the same issue ? Thanks in advance.

**Comment by momur** [comment]
Hello, 

I attempted installing celloracle on my local pc (Mac, M1) but it didnt work for so many reasons. I was only able to manage installing it on HPC servers. 

hope that helps

**Comment by DrNamwob** [comment]
Were you able to get it working? I have an Apple M1 and finally got things working after quite a struggle.  I can help if you're still working through this.

**Comment by momur** [comment]
No, I couldn't make it work on my pc. I installed it on servers, working on there. But I would love to hear if you have tips to share?

**Comment by DrNamwob** [comment]
Sure, I can try to help. There were several issues, but some the main ones included installing velocyto, gimmemotifs, and then celloracle itself. 

I had to use a different compiler to install velocyto -- G++. For gimmemotifs, I installed it from source, but had to go in and modify one of the files because it wasn't formatted properly.  I also installed celloracle from source and had to change the requirements.txt file to not include gimmemotifs (because I already installed it directly). There were several subissues along the way, but those were relatively easy to fix, like installing the specific versions of various packages.  I hope this helps! 

If you have any specific issues, please feel free to reach out via email. I see that you're also a PhD student/candidate. bowmand8@msu.edu

**Comment by Jovo-RNA** [comment]
How did you install them exactly?
